CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+## Overview
+
+This documents explains the processes and practices recommended for contributing enhancements to this operator.
+
+- As this is a private repository containing the charm, consult the JAAS/Livepatch team
```

Comment:
hmm.. we'll have to rephrase this.. i think this is not to be a private repository..